recipes/r-cba/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = '0.2-19' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-cba
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: cba_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/cba_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/cba/cba_{{ version }}.tar.gz
+
+  sha256: 54686d67b6b43b27c4143a593f6669fc0cc661188f91fed80b7be6220fd3da5b
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-proxy
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
+    - r-proxy
+    - libgcc  # [not win]
+
+test:
+  commands:
+    - $R -e "library('cba')"  # [not win]
+    - "\"%R%\" -e \"library('cba')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=cba
+  license: GPL-2
```

Comment:
It seems you have not used the latest version.
We are trying to add https://github.com/bgruening/conda_r_skeleton_helper/blob/master/run.sh#L22 to the recipes :(

Reply:
Oh I see. I'll change that asap :-)